Source/Atomic/IO/Log.cpp

```diff
@@ -228,10 +228,10 @@ void Log::WriteRaw(const String& message, bool error)
     if (logInstance->quiet_)
     {
         if (error)
-            __android_log_print(ANDROID_LOG_ERROR, "Urho3D", message.CString());
+            __android_log_print(ANDROID_LOG_ERROR, "Atomic", "%s", message.CString());
     }

```

Comment:
Upstream has this [fix](https://github.com/urho3d/Urho3D/blob/master/Source/Urho3D/IO/Log.cpp#L231) already 🐫